lambdas/handlers/token_handler.py

```diff
@@ -18,6 +19,7 @@
 @ensure_environment_variables(
     names=["AUTH_STATE_TABLE_NAME", "AUTH_SESSION_TABLE_NAME"]
 )
+@handle_lambda_exceptions
 def lambda_handler(event, context):
     request_context.app_interaction = LoggingAppInteraction.LOGIN.value
 
```

Comment:
Suggest to add an entry point log just for consistency

```suggestion
    logger.info("Token request handler triggered")
```